schema/base_schema.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel
+from typing import Optional
+
+
+class BaseSchema(BaseModel):
+    latitude: float
+    longitude: float
+    priority: Optional[str]
+
+    class Config:
+        orm_mode = True
```

Comment:
1. To avoid misunderstanding rename the file to drone_schemas.py or drone_task.py - all schemas to one model or group models should be placed in one file.
2. Rename the `BaseSchema` class to the `DroneTaskBase` class.
3. Do you need to send `priority` to a drone at all?
4. Add the schemas from the drone_schema and admin_schema files to this file.